openreview/conference/builder.py

```diff
@@ -1541,13 +1544,19 @@ def is_release_authors(is_note_accepted):
                 if decision:
                     venue += ' ' + decision
                 submission.content['venueid'] = venueid
-                submission.content['venue'] = venue
+                submission.content['venue'] = venue.strip()
```

Comment:
you can use the function `decision_to_venue` here and we avoid duplicate code.

Reply:
you can pass `note_accepted` to `decision_to_venue` so the implementation doesn't need to check again `if 'Accept' in...`, then:

```
submission.content['venue'] = decision_to_venue(decision_note.content['decision'], note_accepted)
```

Reply:
I think this won't work because when you are translating the map you don't have `note_accepted`. I think you can have another function `is_accepted(decision)` that is being called from `decision_to_venue` and to set `note_accepted`.

Reply:
done